shenyu-plugin/shenyu-plugin-request/src/test/java/org/apache/shenyu/plugin/request/handler/RequestPluginHandlerTest.java

```diff
@@ -0,0 +1,78 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.plugin.request.handler;
+
+import org.apache.shenyu.common.dto.RuleData;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * request plugin handler test.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class RequestPluginHandlerTest {
+    
+    private RequestPluginHandler requestPluginHandler;
+    
+    private RuleData ruleData;
+    
+    @Before
+    public void setUp() {
+        this.requestPluginHandler = new RequestPluginHandler();
+        this.ruleData = new RuleData();
+        this.ruleData.setSelectorId("test-selectorId");
+        this.ruleData.setName("test-request-plugin");
+        this.requestPluginHandler.handlerRule(ruleData);
+    }
+    
+    @Test
+    public void testHandlerRule() {
+        RuleData ruleHandle = mock(RuleData.class);
+        when(ruleHandle.getSelectorId()).thenReturn("selectorId");
+        when(ruleHandle.getName()).thenReturn("ruleName");
+        when(ruleHandle.getHandle()).thenReturn("{\"header\":{\"addHeaders\":{\"addKey\":\"addValue\"},\"replaceHeaderKeys\":{\"oldKey\":\"newKey\"},"
+                + "\"setHeaders\":{\"oldKey\":\"newValue\"},\"removeHeaderKeys\":[\"removeKey\"],\"notEmptyConfig\":true},"
+                + "\"parameter\":{\"addParameters\":{\"addKey\":\"addValue\"},\"replaceParameterKeys\":{\"oldKey\":\"newKey\"},"
+                + "\"setParameters\":{\"oldKey\":\"newValue\"},\"removeParameterKeys\":[\"removeKey\"],\"notEmptyConfig\":true},"
+                + "\"cookie\":{\"addCookies\":{\"addKey\":\"addValue\"},\"replaceCookieKeys\":{\"oldKey\":\"newKey\"},"
+                + "\"setCookies\":{\"oldKey\":\"newValue\"},\"removeCookieKeys\":[\"removeKey\"],\"notEmptyConfig\":true},\"emptyConfig\":false}");
+        
+        this.requestPluginHandler.handlerRule(ruleHandle);
+    }
+    
+    @Test
+    public void testRemoveRule() {
+        RuleData ruleData = new RuleData();
+        ruleData.setSelectorId("test");
+        ruleData.setName("test");
+        this.requestPluginHandler.removeRule(this.ruleData);
+        this.requestPluginHandler.removeRule(ruleData);
+        this.requestPluginHandler.removeRule(null);
```

Comment:
You're just calling methods. I wonder what are you testing here?

Reply:
I fixed this, please review  , tks

Reply:
> You're just calling methods. I wonder what are you testing here?

I fixed this, please review , tks